inbm/Changelog.md

```diff
@@ -10,7 +10,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
  - RTC 536601 - Added 'source' command to INBM. This command manages `/etc/apt/sources.list` and `/etc/apt/sources.list.d/*` and associated gpg keys on Ubuntu.
+- RTC 537769 -  Added verification of GPG key URIs against a list of trusted repositories for enhanced security
```

Comment:
The changelog entry for RTC 537769 should be more descriptive. Instead of just stating "Added verification of GPG key URIs," it should explain the impact, such as "Enhanced security by verifying GPG key URIs against a list of trusted repositories before adding them." [medium]